### PR TITLE
Add unit tests for delete notes and add notes command parser

### LINKED ISSUE #243: Add unit tests for delete notes

### PR DESCRIPTION
**Pull request by ITLimJiaWei**
Closes #243

### CODE REVIEW

File: src/test/java/bizbook/logic/commands/DeleteNotesCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        // Mock the behavior of modelMock to return personList when needed
+        when(modelMock.getFilteredPersonList()).thenReturn(personList);
+
+        // Mock focused person property
+        ObjectProperty<Person> focusedPersonProperty = new SimpleObjectProperty<>(validPerson);
+        when(modelMock.getFocusedPerson()).thenReturn(focusedPersonProperty);
+
+        // Set up the person index and note index
+        Index validPersonIndex = INDEX_FIRST_PERSON;
+        Index validNoteIndex = Index.fromOneBased(1);
```
Comment (Nicholas-Cheng-De-Fei) [inline src/test/java/bizbook/logic/commands/DeleteNotesCommandTest.java:59]: I think this one you should add it as the `INDEX_FIRST_PERSON` as what was done in the line above for reusability.

So like the first person for the test address book will have 2 notes

File: src/test/java/bizbook/logic/commands/DeleteNotesCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        assertThrows(CommandException.class, Messages.MESSAGE_INVALID_NOTE_INDEX, ()
+                -> deleteNotesCommand.execute(modelMock));
+    }
+
+    @Test
+    public void equals() {
+        Index indexFirstPerson = INDEX_FIRST_PERSON;
+        Index indexSecondPerson = INDEX_SECOND_PERSON;
+        Index noteIndexFirst = Index.fromOneBased(1);
+        Index noteIndexSecond = Index.fromOneBased(2);
```
Comment (Nicholas-Cheng-De-Fei) [inline src/test/java/bizbook/logic/commands/DeleteNotesCommandTest.java:132]: Same as what was mentioned above

**Review by Nicholas-Cheng-De-Fei — CHANGES_REQUESTED** [review]

**Review by Nicholas-Cheng-De-Fei — APPROVED** [review]
LGTM

**Review by sheenkerr — APPROVED** [review]
LGTM

**Pull request merged**